docs/integrations-guides/custom_security.md

```diff
@@ -0,0 +1,73 @@
+- Maintains a mapping of activation block number (ABN) → `EigenDACertVerifier` address: new verifiers can be added at any future block
+- Routes `checkDACert` requests to the appropriate EigenDACertVerifier contract based on reference block number (RBN) in the abi.encoded cert
+
+Note that the Router is only compatible with EigenDACertVerifiers versions >= 3.
+
+Each rollup stack will need to integrate the Router in different ways to maintain a secure integration. We refer users to their respective rollup stack docs:
+- [OP Stack Integration](./rollup-guides/op-stack/README.md)
+- [Arbitrum Orbit Integration](./rollup-guides/orbit/overview.md)
+- [zkSync ZK Stack Integration](./rollup-guides/zksync/README.md)
+
+## Quick Start
+
+The flow to get setup with a CertVerifierRouter and enabling custom quorums would typically look like:
+1. Rollup integrates with EigenDA via the [proxy](./eigenda-proxy/eigenda-proxy.md), using the network-default [EigenDACertVerifier](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.8.1) deployed by EigenDA.
+2. Rollup decides it wants a custom quorum, or to change its security thresholds.
+3. Rollup deploys its own EigenDACertVerifierRouter contract, [initializing](https://github.com/Layr-Labs/eigenda/blob/5fa94eab470823c8fdd829fc5974fe2969068a21/contracts/src/periphery/cert/router/EigenDACertVerifierRouter.sol#L41) it with the same EigenDACertVerifier address it's been using as its ABN=0 verifier. The following deployment [script](https://github.com/Layr-Labs/eigenda/tree/master/contracts/script/deploy/router) can be used for this.
+4. Rollup operators restart their [eigenda-proxy](./eigenda-proxy/eigenda-proxy.md) instance, changing the [--eigenda.v2.cert-verifier-router-or-immutable-verifier-addr](https://github.com/Layr-Labs/eigenda-proxy/blob/34ff55add522a9c7ade281919efa46cd9ac78ce1/docs/help_out.txt#L56) flag to point to the router.
+5. Rollup deploys a new EigenDACertVerifier contract with [_quorumNumbersRequired](https://github.com/Layr-Labs/eigenda/blob/5fa94eab470823c8fdd829fc5974fe2969068a21/contracts/src/periphery/cert/EigenDACertVerifier.sol#L43) including its custom quorum number.
+6. Rollup calls [addCertVerifier](https://github.com/Layr-Labs/eigenda/blob/5fa94eab470823c8fdd829fc5974fe2969068a21/contracts/src/periphery/cert/router/EigenDACertVerifierRouter.sol#L44) on the router, activating the new cert verifier at any future block number of its choice.
```

Comment:
worth noting that the update won't be reflected until disperser chooses rbn >= future activation block number